data/products.yaml

```diff
@@ -29,3 +29,5 @@ kubecarrier:
       name: master
     - release: v0.1
       name: v0.1
+    - release: v0.2
+      name: v0.2
```

Comment:
If these were sorted before the 0.1 release, 0.2 would be the new default version to show.